### PR TITLE
P24 and EPS Bank account elements

### PR DESCRIPTION
**Pull request by michaelc-stripe**
### Summary & motivation
- Add components and types for `p24Bank` and `epsBank` elements.
 - Add an example for each

### API review

Copy [this template] **or** link to an API review issue.

[this template]:
  https://github.com/stripe/react-stripe-js/tree/master/.github/API_REVIEW.md

### Testing & documentation
Manually tested examples in storyboard

### CODE REVIEW

**Review by christopher-stripe — COMMENTED** [review]
Looks great. 

Since this depends on the new types introduced in https://github.com/stripe/stripe-js/pull/134, we'll have to publish those changes first. Once we do, then we can update the [@stripe/stripe-js peer dependency version](https://github.com/stripe/react-stripe-js/blob/1ebc3c4764f295cbb08107ed92de38412b3375ec/package.json#L110) and merge this.

**Comment by cjavilla-stripe** [comment]
lurking here :) 

Looks like the docs use the `EpsBankElement` naming convention rather than `EPSBankElement`. I think that's consistent with what we do for SEPA with SEPA with `IbanElement` and the names for the confirm methods.

**Comment by christopher-stripe** [comment]
> Looks like the docs use the EpsBankElement naming convention rather than EPSBankElement. I think that's consistent with what we do for SEPA with SEPA with IbanElement and the names for the confirm methods.

Thanks @cjavilla-stripe! Agreed that it should be `EpsBankElement`. 

@michaelc-stripe I just published [v1.12.0](https://github.com/stripe/stripe-js/releases/tag/v1.12.0) of `@stripe/stripe-js`. Could you update the peer dep in this PR and then tag me for a review again?

**Comment by michaelc-stripe** [comment]
@christopher-stripe Just to confirm my own understanding, it sounds like @cjavilla-stripe is advocating for a naming of `EPSBankElement` and not `EpsBankElement` (meaning we should in-fact change the docs)? This would make sense to me as 'EPS' is an acronym like SEPA, and is branded in all-caps

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (christopher-stripe) [inline .idea/codeStyles/Project.xml:1]: Is everything committed in the `.idea/` folder here useful for other devs? Or is it configuration unique to your setup? I'm wondering whether we should add `.idea` to the `.gitignore` file and not commit all this.

**Review by christopher-stripe — APPROVED** [review]
Quick question about the `.idea` files, but looks great!

**Pull request merged**